src/client.rs

```diff
@@ -400,15 +410,17 @@ impl Client {
     /// instead of panicking.
     pub fn new() -> Client {
         ClientBuilder::new()
+            .use_sys_proxy()
```

Comment:
Hi, @seanmonstar , would you please reiview that put use_sys_proxy() here okey?

Reply:
I think `Client::new` should use the defaults of the builder, so that `Client::new()` and `Client::builder().build()` create the same client. Also, it *might* be that people have proxies setup in their environment, but don't expect reqwest to use them, so I believe it's best if the default is **off**, and we can switch to default **on** in the next breaking change. Perhaps [the library documentation](https://docs.rs/reqwest/0.9.*/reqwest/#proxies) should mention that...

Reply:
Sorry, my fault for doesn't catch your idea before.  I have remove it for now.